### PR TITLE
help output for buildenv scripts 

### PR DESCRIPTION
**Pull request by daschuer**
This may ... help

### CODE REVIEW

File: tools/debian_buildenv.sh [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
 
+CALL=$0
 COMMAND=$1
 shift
 
 case "$COMMAND" in
     name)
-        echo "No build environment needed for Ubuntu, please install dependencies using apt." >&2
+        echo "No build environment name required for Debian based distros." >&2
+        echo "This script installs the build bependeccies via apt using the \"setup\" option." >&2
```
Comment (ronso0) [inline tools/debian_buildenv.sh:13]: typooo

Comment (daschuer) [reply tools/debian_buildenv.sh:13]: I have rebased a fix

File: tools/debian_buildenv.sh [hunk 2]
```diff
@@ -3,12 +3,14 @@
 # shellcheck disable=SC1091
 set -o pipefail
 
+CALL=$0
```
Comment (Holzhaus) [inline tools/debian_buildenv.sh:6]: Do we really need to create an alias for $0?

**Review by Holzhaus — APPROVED** [review]

File: tools/debian_buildenv.sh (hunk 2 above)
Comment (daschuer) [reply tools/debian_buildenv.sh:6]: Because of the following shift call.
Do you know why it is there?

File: tools/debian_buildenv.sh (hunk 1 above)
Comment (ronso0) [reply tools/debian_buildenv.sh]: thx

File: tools/debian_buildenv.sh (hunk 2 above)
Comment (Holzhaus) [reply tools/debian_buildenv.sh:6]: Possibly a leftover from a prevous version of the code.

Comment (daschuer) [reply tools/debian_buildenv.sh:6]: should we try to remove it with this PR?

Comment (Holzhaus) [reply tools/debian_buildenv.sh:6]: Yes, please try

**Comment by daschuer** [comment]
Still all green without shift. Merge?

**Pull request merged**